app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(list_of_players: list[Player]) -> int:
+    return sum(player.get_rating() for player in list_of_players)
+
+
+def elves_concert(list_of_elves: list[Elf]) -> None:
+    [elf.play_elf_song() for elf in list_of_elves]
```

Comment:
no need to use list comprehension as you don't need to save the result of play_elf_song(). a simple loop is better